web_ui/frontend/app/cache/page.tsx

```diff
@@ -83,16 +84,25 @@ export default function Home() {
                                     },
                                 }}
                                 datasetOptions={[
-                                    {label: "Total Storage (Bytes)", borderColor: "#000000"},
-                                    {label: "Free Storage (Bytes)", borderColor: "#54ff80"}
+                                    {label: "Total Storage (Gigabytes)", borderColor: "#000000"},
+                                    {label: "Free Storage (Gigabytes)", borderColor: "#54ff80"}
                                 ]}
+                                datasetTransform={(dataset) => {
+                                    dataset.data = dataset.data.map(p => {
+                                        let {x, y} = p as DataPoint
+                                        y = y / (10**9)
```

Comment:
10^9 or 1024*1024*1024? 🤣

Reply:
uhh. The astra was eaten up by markdown. You know what I mean

Reply:
Ah a Gibibyte connoisseur. Call me crazy but I think people are more familiar with decimal Gigabytes over binary.